windows/gregorio.iss

```diff
@@ -38,7 +38,8 @@ Source: "../CHANGELOG.md"; DestDir: "{app}";
 Source: "../README.md"; DestDir: "{app}";
 Source: "../CONTRIBUTORS.md"; DestDir: "{app}";
 Source: "../UPGRADE.md"; DestDir: "{app}";
-Source: "../doc/UserManual.pdf"; DestDir: "{app}";

```

Comment:
to be more consistent with other comments, maybe `PARSE_VERSION_FILE_NEXTLINE` would be better?